README.md

```diff
@@ -8,6 +8,7 @@ Friendship Game.  For Vote, Rank,  QA ...
 npm install
 npm start
 ```
+open [http://localhost:8080/](http://localhost:8080/)
```

Comment:
We could also document that `` npm start -- --open `` opens this URL in the browser when the server is ready.